config/flog.yml

```diff
@@ -1,2 +1,2 @@
 ---
-threshold: 27.7
+threshold: 33.8
```

Comment:
Thats a more than 25% increase, I think there is more duplication to fight to remove some AST complexity till we can merge this. There are unaddressed flags likely to contribute:

* https://github.com/mbj/mutant/pull/565/files#r61767066 (with some instances)